packages/pebble-shared/icons.json

```diff
@@ -54,6 +54,7 @@
   "copy-filled",
   "copy",
   "cp-icon",
+  "cross",
```

Comment:
we have `close`, how about `close-bold`?

Reply:
new icons follow material design (mdc). i want to use mdc names and use suffixes for variants.

- outlined (base design)
- filled
- rounded
- sharp
- toned

in mdc, this icon would be "[close][1]", but pebble already has "close" icon and it is visually different

i named it "cross" bcoz it resembles the geometry and [mdc doesn't have "cross"][2] 

[1]: https://fonts.google.com/icons?selected=Material+Icons+Outlined:close
[2]: https://fonts.google.com/icons?icon.query=cross